.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+# Check if code is formatted
+if ! npx prettier@3.0.3 --check . ; then
+  echo "Code is not formatted. Run 'npx prettier@3.0.3 --write .' to format before committing."
```

Comment:
A last thing, `npx prettier@3.0.3 --write .` should be replaced by `npm run format:json` as well

Reply:
indeed, ✅